runtime/oti/j9javaaccessflags.h

```diff
@@ -99,6 +99,7 @@
  * See map in ROMClassBuilder::computeExtraModifiers for
  * available slots.
  */
+#define J9AccClassIsShared 0x20
```

Comment:
Please add the new flag into the comment here:

https://github.com/eclipse-openj9/openj9/blob/0c6cdfd79fe3f612c6bb80ba72bd9a02bfa13ca7/runtime/bcutil/ROMClassBuilder.cpp#L1219

Reply:
I updated the comment.